docs/core/additional-tools/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,45 @@
+ # Using Microsoft Xml Serializer Generator on .NET Core
+
+Like the Xml Serializer Generator (sgen.exe) on desktop, Microsoft.XmlSerializer.Generator NuGet package is the solution for .NET Core and .NET Standard Libraries. It creates an Xml serialization assembly for types contained in an assembly to improve the startup performance of Xml serialization when serializing or de-serializing objects of those types using XmlSerializer. 
```

Comment:
is the solution for what?

.NET Standard Libraries -> .NET Standard projects

link to the API as shown for index.md

Reply:
the solution equivalent to sgen.exe on desktop.